vermouth/molecule.py

```diff
@@ -424,6 +424,7 @@ def copy(self):
         """
         new = self.subgraph(self.nodes)
         new.name = self.name
+        new.citations = self.citations
```

Comment:
Since these are both in the subgraph method I think they can both be removed here.